packages/fluentui/react-northstar/package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "@fluentui/a11y-testing": "^0.1.0",
     "@fluentui/eslint-plugin": "^1.0.0-beta",
-    "@fluentui/react-conformance": "^1.0.0",
+    "@fluentui/react-conformance": "^1.0.0-beta.0",
```

Comment:
Can you change the version to `0.2.0` (the package itself and dependencies)?

Reply:
Done